pkgs/applications/version-management/git-and-tools/git-team/default.nix

```
@@ -0,0 +1,25 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+buildGoModule rec {
+  pname = "git-team";
+  version = "1.7.0";
+
+  src = fetchFromGitHub {
+    owner = "hekmekk";
+    repo = "git-team";
+    rev = "v${version}";
+    sha256 = "0nl5j64b61jw4bkf29y51svjbndmqqrqx96yaip4vjzj2dx9ywm4";
+  };
+
+  vendorSha256 = null;
+
+  proxyVendor = true;
+
+  meta = with lib; {
+    description =
+      "Git-team provides a command line interface for managing and enhancing git commit messages with co-authors.";
```

```suggestion
      "Command line interface for managing and enhancing git commit messages with co-authors";
```

See [Submitting Changes](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#submitting-changes).

---

Lesson learned. :D